OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -126,9 +129,12 @@ public LaserZap(LaserZapInfo info, ProjectileArgs args, Color color)
 
 		public void Tick(World world)
 		{
+			// Position of firing actor might have changed
+			source = args.Source;
```

Comment:
If agreed, alternatively this commit 'Update and use cached source pos in LaserZap' could be removed, as previously args.Source was used and this new issue could be solved for this/other traits if needed in another PR?